<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameOverWidget</class>
 <widget class="QWidget" name="GameOverWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameOverWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lbl3">
     <property name="text">
      <string>平均反应时间:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="TotalFire">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLabel" name="HitRate">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="TotalGameTime">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="TotalEnemyNum">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl1">
     <property name="text">
      <string>游戏类型:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl2">
     <property name="text">
      <string>总分数:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lbl4">
     <property name="text">
      <string>总出现敌人数量:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="TotalKillNum">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="AverageReactTime">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lbl5">
     <property name="text">
      <string>总消灭敌人数量:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lbl8">
     <property name="text">
      <string>命中率:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lbl6">
     <property name="text">
      <string>总射击数量:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lbl7">
     <property name="text">
      <string>本局游戏时间:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="GameType">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="TotalScore">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
